psa-crypto/src/types/key.rs

```diff
@@ -390,6 +469,23 @@ pub struct UsageFlags {
     pub derive: bool,
 }
 
+impl Default for UsageFlags {
+    fn default() -> Self {
+        UsageFlags {
+            export: false,
+            copy: false,
+            cache: false,
+            encrypt: false,
+            decrypt: false,
+            sign_message: false,
+            verify_message: false,
+            sign_hash: false,
+            verify_hash: false,
+            derive: false,
+        }
+    }
+}
```

Comment:
Is this not what deriving `Default` would obtain?

Reply:
Oh yes, you are right, I forgot that you can derive `Default`